content/en/docs/started/k8s/overview.md

```diff
@@ -4,10 +4,7 @@ description = "Instructions for installing Kubeflow on your existing Kubernetes
 weight = 1
                     
 +++
-{{% alert title="Out of date" color="warning" %}}
-This guide contains outdated information pertaining to Kubeflow 1.0. This guide
-needs to be updated for Kubeflow 1.1.
-{{% /alert %}}
+
```

Comment:
@swiftdiaries Hi! Let us know if anything should be done on our side (especially since v1.2 works are underway) and/or you have any updates regarding this PR. Have a good weekend 👍

Reply:
I don't really have any updates.

Reply:
Ok @swiftdiaries 👍  How about these? https://github.com/kubeflow/website/pull/2195#discussion_r488932386 Sorry, I can't edit anything here, only propose suggestions

Reply:
We've been waiting for a reply to this https://github.com/kubeflow/website/pull/2195#discussion_r488932386, basically.

Also, the KF on K8s overview page - https://www.kubeflow.org/docs/started/k8s/overview/ - has had some updates. Can you please resolve the merge conflict (as per https://prow.k8s.io/pr?query=is%3Apr+repo%3Akubeflow%2Fwebsite+author%3Aswiftdiaries+head%3Aupdate-min-requirement)? cc @RFMVasconcelos

Reply:
Can I close this PR and someone else who's working on the k8s config can possibly create a new one?
I don't have the context required to make the changes.

Reply:
No worries @swiftdiaries 👍 

The outstanding thing appears to be to do with DVP by adding:

```
* A default Storage Class with a Dynamic Volume Provisioner.
   - Multiple components like Kubeflow Pipelines and Jupyter Web App assume the existence of a volume provisioner.
```

or

```
You need a default
[StorageClass](https://kubernetes.io/docs/concepts/storage/storage-classes/)
with a
[dynamic volume provisioner](https://kubernetes.io/docs/concepts/storage/dynamic-provisioning/).
For more information, refer to
[this guide](https://www.kubeflow.org/docs/started/k8s/kfctl-k8s-istio/#before-you-start).
```

Can you please confirm? 

Thanks for your help

Reply:
The second one probably has more fleshed out details

Reply:
And is better from an end-user perspective